src/my-workspace/views/MyWorkspace.tsx

```diff
@@ -116,31 +115,50 @@ const MyWorkspace: FunctionComponent<MyWorkspaceProps> = ({ collections, history
 		}
 	};
 
-	return (
-		<Fragment>
-			<Container background="alt" mode="vertical" size="small">
-				<Container mode="horizontal">
-					<h2 className="c-h2 u-m-0">Mijn Werkruimte</h2>
+	const renderTabsAndContent = (data: {
+		app_collections_aggregate: { aggregate: { count: number } };
+	}) => {
+		const counts = {
```

Comment:
Gaat het niet makkelijker zijn om deze ergens globaler te definiëren in de component ipv ze telkens mee te geven aan de functies waar ze nodig zijn?

Reply:
Ja dat had ik eerst geprobeerd, maar door die graphql query is dat wat raar
want dan moet je state setten in de renderTabsAndContent method
We zouden nog een manier moeten hebben om die call te doen als effect en dan enkel de render opnieuw uit te voeren als de count aangepast zijn.
https://github.com/apollographql/apollo-client/issues/4427

Als je wil mag je dit omvormen, maar mss moeten we dit in een separate PR doen?

Reply:
Op zich niet erg om hier state te zetten denk ik, laat het maar zo voorlopig